repotracker/repotracker_test.go

```diff
@@ -328,7 +330,7 @@ tasks:
 	assert.NoError(t, d.Insert())
 
 	p := &model.Project{}
-	ctx := context.Background()
+	ctx = context.Background()
```

Comment:
This assignment shouldn't be needed since the wrapped context on L269 is already using the background context. The `context.WithCancel(context.Background())` wrapping at the top is only used to ensure that any context-dependent operations that were running in the test (e.g. goroutines) will stop when the main goroutine (i.e. the one running this test) reaches the last line of this test.